ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -45,14 +49,21 @@ def _es_modules():
             pass
 
 
+def get_version():
+    import elasticsearch
```

Comment:
`_patch(elasticsearch)` already takes in an imported `elasticsearch` so we don't need to import it twice.
```suggestion
def get_version(elasticsearch):
```

Reply:
ah thanks, I got mixed up!